composer.json

```diff
@@ -1,6 +1,15 @@
 {
+    "require": {
+        "krokedil/settings-page": "^1.1.0"
+    },
```

Comment:
We should use scoping for this and not add it directly to the composer file, rather implement it using the wpify scoper setup we have in other plugins as well.

Reply:
An attempt to add wpify scoping pushed :)

I'm still not seeing the tabs (using an existing json file), so I'm sure something is still missing.